gui-apps/foot/foot-1.11.0.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson xdg
+
+DESCRIPTION="Fast, lightweight and minimalistic Wayland terminal emulator"
+HOMEPAGE="https://codeberg.org/dnkl/foot"
+SRC_URI="https://codeberg.org/dnkl/foot/archive/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+grapheme-clustering"
+
+DEPEND="
+	grapheme-clustering? (
+		dev-libs/libutf8proc:=
+		>=media-libs/fcft-3.0.0[harfbuzz]
+	)
+	dev-libs/wayland
+	>=media-libs/fcft-3.0.0
+	media-libs/fontconfig
+	x11-libs/libxkbcommon
+	x11-libs/pixman
+	dev-libs/wayland-protocols
+	dev-libs/tllist
```

Comment:
These two are DEPEND-only and shouldn't be in RDEPEND.

May want to set COMMON_DEPEND since you have some exclusives to both.